_episodes/03-ncbi-sra.md

```diff
@@ -13,18 +13,32 @@ keypoints:
+There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are provided by the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) has the [Sequence Read Archive (SRA)](https://trace.ncbi.nlm.nih.gov/Traces/sra/) and the [European Nucleotide Archive (EMBL-EBI)](https://www.ebi.ac.uk/) has their [Nucleic Acid Archive](https://www.ebi.ac.uk/ena). In this lesson we're working with the NCBI database, but the general process is the same for any database.
 
 # Accessing the original archived data
 
-The [sequencing dataset (from Tenaillon paper) adapted for this lesson](http://www.datacarpentry.org/organization-genomics/data/) was obtained from the [NCBI Sequence Read Archive](http://www.ncbi.nlm.nih.gov/sra) which is a large (~27 petabasepairs/2.7 x 10^16 basepairs as of April 2019 ([source](https://www.ncbi.nlm.nih.gov/sra/docs/sragrowth/))) repository for next-generation sequence data. Like many NCBI databases, it is complex and mastering its use is greater than the scope of this lesson. Very often, as in the Blount paper, there will be a direct link (perhaps in the supplemental information) to where on the SRA the dataset can be found. E.g. the link from the Tenaillon paper is [http://www.ncbi.nlm.nih.gov/sra?term=SRA026813](http://www.ncbi.nlm.nih.gov/sra?term=SRA026813)  
+The [sequencing dataset (from Tenaillon, *et al.* 2016) adapted for this lesson](http://www.datacarpentry.org/organization-genomics/data/) was obtained from the [NCBI Sequence Read Archive](http://www.ncbi.nlm.nih.gov/sra) which is a large (~27 petabasepairs/2.7 x 10^16 basepairs as of April 2019) repository for next-generation sequence data. Like many NCBI databases, it is complex and mastering its use is greater than the scope of this lesson. Very often there will be a direct link (perhaps in the supplemental information) to where the SRA dataset can be found. We are only using a small part of these data, so a direct link cannot be found. If you have time, go through the following detailed description of finding the data we are using today (otherwise skip to the next section). 
 
 ## Locate the Run Accessory Table for the Lenski Dataset on the SRA
 
+See the figures below for how data access was placed within the original paper. 
+
+<img style='border:1px solid #000000' src="../fig/03_paper_header.png" width="800"/>
+
+The **above image** shows the title of the study, as well as the authors.
+
+The **image below** shows an excerpt from the paper where the text for locating accession information was reported. In this case, the text was located after the reference section.
```

Comment:
The **image below** shows an excerpt from the paper that includes information on how to locate the sequence data. In this case, the text is placed after the reference section.